initdb.py

```diff
@@ -83,7 +83,8 @@ def create_database(cursor):
         print("Creating database {}".format(DB_NAME))
         cursor.execute(create_sda_db)
     except mysql.connector.Error as mysql_error:
-        raise Exception('Failed creating database: {}'.format(mysql_error))
+        raise MySqlError(message='Failed creating database: {}'.format(mysql_error),
```

Comment:
Don't put `mysql_error` in the message. The end-user doesn't need to see the stacktrace or original error. That's why we have `args`.